packages/components/psammead-bulletin/src/index.test.jsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
-import { latin } from '@bbc/gel-foundations/scripts';
+import { latin, arabic } from '@bbc/gel-foundations/scripts';
 import Image from '@bbc/psammead-image';
 import Bulletin from '.';
 
 /* eslint-disable react/prop-types */
-const BulletinComponent = ({ script, service, isLive, mediaType, ctaText }) => {
+const BulletinComponent = ({
+  script,
+  service,
+  isLive,
+  mediaType,
+  ctaText,
+  dir = 'ltr',
+  lang = null,
```

Comment:
We can remove now this
```suggestion

```

Reply:
https://github.com/bbc/psammead/pull/3430#discussion_r416735056